distributed/scheduler.py

```diff
@@ -4201,7 +4201,7 @@ def update_data(
                 self.client_desires_keys(keys=list(who_has), client=client)
 
     def report_on_key(self, key: str = None, ts: TaskState = None, client: str = None):
-        assert (key is None) + (ts is None) == 1, (key, ts)
+        assert (key is None) != (ts is None), (key, ts)
```

Comment:
Found another case in the scheduler like this, addressing in PR ( https://github.com/dask/distributed/pull/4342 ).